index.js

```diff
@@ -33,12 +33,15 @@ module.exports = options => {
 	options.directory = path.resolve(electron.app.getAppPath(), options.directory);
 
 	const handler = async (request, callback) => {
-		const indexPath = path.join(options.directory, 'index.html');
 		const filePath = path.join(options.directory, new URL(request.url).pathname);
 
-		callback({
-			path: (await getPath(filePath)) || indexPath
-		});
+		const resolvedPath = await getPath(filePath);
+
+		if (resolvedPath) {
+			callback({path: resolvedPath});
+		} else {
+			callback({error: -6});
```

Comment:
Where did you get this number from? It should be moved into a variable with a descriptive name.

Reply:
It's documented here 
[RegisterFileProtocol](https://github.com/electron/electron/blob/master/docs/api/protocol.md#protocolregisterfileprotocolscheme-handler-completion)
[net error list](https://cs.chromium.org/chromium/src/net/base/net_error_list.h)

Found a similar implementation here [electron-routes](https://github.com/MarshallOfSound/electron-routes/blob/4240bffd875be38dc1538de7db27d0bd84e18f2f/src/Router.js#L84)